tests/testDipoleFitter.py

```diff
@@ -220,12 +238,14 @@ def testDipoleTask(self):
         return result
 
     def testDipoleEdge(self):
-        """Test the dipole fitting singleFramePlugin. Test that the dipoles
-        which are too close to the edge raise the correct exception.
+        """Test the edge case for dipole fitting singleFramePlugin.
+
+        Test that the dipoles which are too close to the edge are
+        indicated as so in the catalog.
```

Comment:
What does "edge case" mean here?

I feel like this needs a longer docstring, as I can't actually tell what it's testing from the code. Is an assert that DipoleFit_flag is set (meaning this was found as a dipole) really all we need to test? The test should probably also test that the correct number of dipoles were found.